gpu.Dockerfile

```diff
@@ -9,8 +9,6 @@ ADD clean-layer.sh  /tmp/clean-layer.sh
 COPY --from=nvidia /etc/apt/sources.list.d/cuda.list /etc/apt/sources.list.d/
 COPY --from=nvidia /etc/apt/sources.list.d/nvidia-ml.list /etc/apt/sources.list.d/
 COPY --from=nvidia /etc/apt/trusted.gpg /etc/apt/trusted.gpg.d/cuda.gpg
-# See b/142337634#comment28
-RUN sed -i 's/deb https:\/\/developer.download.nvidia.com/deb http:\/\/developer.download.nvidia.com/' /etc/apt/sources.list.d/*.list
```

Comment:
This was a workaround to download the packages over `http`. 

However, NVIDIA servers forces `https` now.

Removing the work-around now that we can fetch package over https with `apt`.